common/changes/@cadl-lang/compiler/azhang_remove_azure_init_2023-01-12-23-05.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cadl-lang/compiler",
+      "comment": "undo `azure` alias for cadl init. Will continue to rely on URL.",
```

```suggestion
      "comment": "",
```

We don't need a release note since this never shipped.